packages/react-scripts/per-lang-loader/PerLangPlugin.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const dependencyTree = require('dependency-tree')
+const jsonfile = require('jsonfile')
+const path = require('path')
+const fs = require('fs')
+const rimraf = require('rimraf')
+
+const builtLocalesDir = `${__dirname}/builtLocales`
+
+class PerLangPlugin {
+  apply(compiler) {
+    const allLocales = {};
+    const manifest = {};
+    // inject a function to loaderContext so loaders can pass back info
+    compiler.hooks.beforeCompile.tapAsync('PerLangPlugin', (params, callback) => {
+      debugger;
+      if (!fs.existsSync(builtLocalesDir)) {
```

Comment:
we're in tapAsync but making sync calls. for manually invoked scrips, sync calls are okay, but for build steps that might be run in parallel, we should probably do this properly async

Reply:
file is deleted now